unlock-app/src/hooks/useFiatChargePrice.ts

```diff
@@ -6,6 +6,8 @@ interface Options {
   tokenAddress?: string
   amount: number
   enabled?: boolean
+  lockAddress?: string
```

Comment:
Since these are optional let's just add that change to +another+ PR to make sure we're not breaking anything?